sap/5.8/modules/ROOT/pages/sap-connector-config-snc-kerberos.adoc

```diff
@@ -0,0 +1,149 @@
+= Configuring an SAP Secure Network Communication Connection with Kerberos
+
+Configure an SAP Secure Network Communication (SNC) with Kerberos to take advantage of security features such as secure data communication between the SAP system client and server, application-level end-to-end security, and the ability to change security products without impacting your SAP business applications. Enabling SNC with Kerberos requires you to configure the client environment, the SAP server, and Anypoint Connector for SAP (SAP Connector). 
+
+NOTE: SAP does not provide support for third-party solutions.
+
+== Before You Begin
+
+To configure an SAP SNC with Kerberos, you must have access to:
+
+* The SAP GUI application. To install this client, Java must already be installed on your machine. Then, download the installer for your operating system from SAP.
```

Comment:
```suggestion
* The SAP GUI application
+
To install this client, Java must already be installed on your machine. Then, download the installer for your operating system from SAP.
```

Reply:
I think, if you have a hanging description, the bullet itself doesn't need the period.